chainguard-dev/edu
# Ensure the user name field is added to Artifactory setup.

Also tidied some of the wording.
Signed-off-by: Max Allan <max.allan@chainguard.dev>

## Type of change
Docs clarification

### What should this PR do?
[Zendesk ticket
](https://chainguardhelp.zendesk.com/agent/tickets/645)

### What are the acceptance criteria? 
Tested as below. Usual checks for style, sanity, etc.

### How should this PR be tested?
Run through the process with a free artifactory. 
Try it with and without a username and see it [not] working.
